### PR TITLE
add oms central to required version matrix

### PR DESCRIPTION
**Pull request by lache-melvin**
Adds required versions of open msupply central for different open msupply remote versions.

It's after 5 and I'm not sure about the wording, let me know if it doesn't make as much sense as it should 😅

### CODE REVIEW

File: content/docs/introduction/requirements.md [hunk 1]
```diff
@@ -108,54 +108,57 @@ Useful productivity accessories for laptop computers include
...
 
 - Install a reputable anti-virus program, including web protection if the computer has access to the internet
 - Disable USB ports - refer to [this site](http://www.thewindowsclub.com/disable-enable-usb-windowunlock-pen-drive-at-office-or-school-computer) for suggested methods
 
 Ensure that access to passwords which allow exceptions to the above two measures are kept secure.
 
-## mSupply
-An installation of mSupply is currently required when running Open mSupply (this is used to manage a number of centralised aspects of the system). 
-The version of both mSupply and Open mSupply is important, as not all versions of each are compatible with each other. 
+## Central Servers
```
Comment (lache-melvin) [inline content/docs/introduction/requirements.md:141]: SORRY formatter field day, only content changes are this section below!

File: content/docs/introduction/requirements.md [hunk 2]
```diff
@@ -108,54 +108,57 @@ Useful productivity accessories for laptop computers include
...
-| Open mSupply         | mSupply               |
-| :--------------- | :-------------------- |
-| 1.1.00 - 1.1.16  | 7.04.01+              |
-| 1.2.00+          | 7.05.05+              |
-| 1.4.00+          | 7.09.00+              |
-| 2.0.00+          | 7.14.04+              |
+From v2.0.00, the Open mSupply central server is also required for successful synchronisation. See the [Open mSupply central server](/docs/getting_started/central/) and [Open mSupply central server site configuration in mSupply](https://docs.msupply.org.nz/synchronisation:sync_sites#open_msupply_central_server_settings) for more details.
 
-If you attempt to connect to an incompatible server you'll get an error message like this:
+You will get the following error message the Open mSupply central server is not configured
```
Comment (roxy-dao) [inline content/docs/introduction/requirements.md:147]:
```suggestion
You will get the following error message if the Open mSupply central server is not configured
```

File: content/docs/introduction/requirements.md [hunk 3]
```diff
@@ -108,54 +108,57 @@ Useful productivity accessories for laptop computers include
...
 
-![server version mismatch!](/docs/introduction/images/version_mismatch.png)
+![v6 not configured!](/docs/introduction/images/v6_not_configured.png)
 
-## Open mSupply central server
+The versions of both central servers and your Open mSupply remote site are important, as not all versions of each are compatible with each other.
 
-From v2.0.00, the Open mSupply central server is required for successful synchronisation. See the [Open mSupply central server](/docs/getting_started/central/) and [Open mSupply central server site configuration in mSupply](https://docs.msupply.org.nz/synchronisation:sync_sites#open_msupply_central_server_settings) for more details. 
+The table below shows which versions of mSupply and Open mSupply Central you will require when running as Open mSupply remote site:
```
Comment (roxy-dao) [inline content/docs/introduction/requirements.md:153]:
```suggestion
The table below shows which versions of mSupply and Open mSupply Central you will require when running as an Open mSupply remote site:
```

File: content/docs/introduction/requirements.md [hunk 4]
```diff
@@ -108,54 +108,57 @@ Useful productivity accessories for laptop computers include
...
-| 1.2.00+          | 7.05.05+              |
-| 1.4.00+          | 7.09.00+              |
-| 2.0.00+          | 7.14.04+              |
+From v2.0.00, the Open mSupply central server is also required for successful synchronisation. See the [Open mSupply central server](/docs/getting_started/central/) and [Open mSupply central server site configuration in mSupply](https://docs.msupply.org.nz/synchronisation:sync_sites#open_msupply_central_server_settings) for more details.
 
-If you attempt to connect to an incompatible server you'll get an error message like this:
+You will get the following error message the Open mSupply central server is not configured
 
-![server version mismatch!](/docs/introduction/images/version_mismatch.png)
+![v6 not configured!](/docs/introduction/images/v6_not_configured.png)
```
Comment (roxy-dao) [inline content/docs/introduction/requirements.md:149]: v6 really needs to be renamed lol

**Review by roxy-dao — APPROVED** [review]
/o/ \o\ /o/ \o\

**Pull request merged**